app/index.js

```diff
@@ -7,7 +7,7 @@ var mb = menubar({
   dir:__dirname, 
   tooltip: "Pomolectron", 
   icon:__dirname + "/res/tomato.png", 
-  width:278, height:250, 
+  width:300, height:250, 
```

Comment:
Any specific reason for changing the dimensions here?

Reply:
The font size for dark mode require extra width. If I let the width to be 278 px, it will break the line, turn the header into strange layout like this : 
![image](https://user-images.githubusercontent.com/1295143/46939224-f5f25400-d08f-11e8-9481-38bbcc56fa5e.png)

Reply:
Okay. Got it.